dht.go

```diff
@@ -231,12 +231,13 @@ func (dht *IpfsDHT) getLocal(key string) (*recpb.Record, error) {
 	log.Debugf("getLocal %s", key)
 	rec, err := dht.getRecordFromDatastore(mkDsKey(key))
 	if err != nil {
+		log.Warningf("getLocal: %v", err.Error())
```

Comment:
Idiom: This can just be `log.Warningf("getLocal: %s", err)`. That way, we'll only turn the error into a string if we actually have warnings enabled.